lua/autorun/server/sv_cfc_contact_forms.lua

```diff
@@ -129,6 +131,16 @@ local function playerCanSubmit( ply )
     return ( playerSubmissionCounts[ply] or 0 ) < 3
 end
 
+local function sendSuccessAlert( ply )
```

Comment:
these could error if the player becomes invalid after the `POST`
wouldn't break anything if they did, but just so you know